packages/testing-kit/src/environment/configuration.ts

```diff
@@ -1,117 +1,61 @@
-import { Network, NetworkGuard, getNetwork } from "@marlowe.io/runtime-core";
-import { formatValidationErrors } from "jsonbigint-io-ts-reporters";
-import { unsafeEither } from "@marlowe.io/adapter/fp-ts";
-import * as E from "fp-ts/lib/Either.js";
-import { Blockfrost } from "lucid-cardano";
-import { SeedPhrase, seedPhrase } from "../wallet/seedPhrase.js";
-import {
-  RestClient,
-  mkRestClient,
-  RuntimeVersion,
-} from "@marlowe.io/runtime-rest-client";
-import * as Lucid from "lucid-cardano";
-import * as G from "@marlowe.io/runtime-rest-client/guards";
-import { MarloweJSON } from "@marlowe.io/adapter/codec";
-
-/**
- * Test Configuration : Read from an env file, it contains :
- * 1. The necessary configuration to run e2e tests over a Lucid Wallet and a Runtime.
- * 2. A Bank Wallet Seed Phrase to provision ephemeral Test Wallets
- */
-export type TestConfiguration = {
-  bank: { seedPhrase: SeedPhrase };
-  lucid: { blockfrost: Blockfrost; node: { network: Lucid.Network } };
```

Comment:
You've re-designed this TestConfiguration type without really using it and discuss with me before... It will impact my current work...

Reply:
This was a first attempt to settup debugging in the tests and I managed to do that. I've barely modified the "testing  infrastructure". Do any change you want on top, make sure it is debuggable still ;). I have made sure that the e2e tests pass before and after my changes